install/prerequisites.adoc

```diff
@@ -390,9 +390,8 @@ the router will be deployed:
 *.cloudapps.example.com. 300 IN  A 192.168.133.2
 ----
 
-In almost all cases, when referencing VMs you must use host names, and the host
-names that you use must match the output of the `hostname -f` command on each
-node.
+When referencing VMs, use the host names that the output of the `hostname -f`
```

Comment:
Why specifically VMs? I don't think there's a difference with physical hosts. "nodes that aren't integrated with a cloud provider" is probably a more precise term

Reply:
The docs specifically refer to VMs in other places: https://docs.openshift.com/container-platform/3.10/install/prerequisites.html#overriding-detected-ip-addresses-host-names

I need to get more info about this change, but I'll try to get feedback from QE on this point. Thanks!

Reply:
Agree, here should not be specifically VMs. 
But the following description is not right. `hostname` == `hostname -f` is not a must or a prerequisites, especially for fresh install. In QE's past testing, in our cluster,  `hostname` is always not equal to `hostname -f`, everything is going well. Of course, if user could assure `hostname` == `hostname -f` would be reduce ratio of hitting some strange issue as much as possible.